BUILD

```diff
@@ -52,7 +52,6 @@ bzl_library(
         "//lib:shell",
         "//lib:structs",
         "//lib:types",
-        "//lib:unittest",
```

Comment:
Drive-by: Since lib.bzl is a stub file, it looks like all these deps can be removed.